module/zfs/zfs_sysfs.c

```diff
@@ -356,38 +356,68 @@ pool_property_show(struct kobject *kobj, struct attribute *attr, char *buf)
  * async calls (i.e. a sync flag). If this mechanism were in place at that
  * time, we could have added a 'channel_program_async' to this list.
  */
-static const char *zfs_features[]  = {
-	/* --> Add new kernel features here (post ZoL 0.8.0) */
-	"initialize",
-	"trim",
+static const char *zfs_kernel_features[]  = {
+	/* --> Add new kernel features here */
+	"com.delphix.vdev_initialize",
```

Comment:
Looks like you accidentally included Don's commit `features.kernel layout should match features.pool`

Reply:
Whoops.   Fixed.